src/content/docs/browser/browser-monitoring/getting-started/introduction-browser-monitoring.mdx

```diff
@@ -38,17 +38,11 @@ Real user monitoring (RUM) gives insights into actual page performance, but you
   src={browserBrowserSummary}
 />
 
-<figcaption>
-  **[one.newrelic.com](https://one.newrelic.com) > Browser > Settings**: Use browser monitoring to see actionable data about your end users' experience with your app.
-</figcaption>
+## Track core web vitals
 
-The [browser monitoring **Summary** page](/docs/browser/browser-monitoring/getting-started/browser-summary-page/) shows important core web vital scores for:
-
-* Largest contentful paint (LCP): how quickly your page content loads
-* First input delay (FID): how quickly the browser can respond to user input
-* Cumulative layout shift (CLS): how unstable is the content being loaded
-
-You can also use the [`PageViewTiming` event](/attribute-dictionary/?event=PageViewTiming) for a more real-time delivery mechanism that does not have dependencies on any other event.
+Google's core web vitals are three metrics to gauge how users experience your browser app - and New Relic can help you track them! There are two places to see core web vitals in New Relic:
+* The [**Summary** page](/docs/browser/browser-monitoring/getting-started/browser-summary-page/) shows core web vitals for your entire site.
+* The [**Web vitals** page](/docs/browser/new-relic-browser/guides/guide-to-core-web-vitals#monitor) shows a breakdown of core web vitals by page URL, device type, location, and more.
```

Comment:
I think that 'page' should go out of the bolded in both cases or maybe remove the word page. What about one of these 2 options?:

- The [**Summary**](/docs/browser/browser-monitoring/getting-started/browser-summary-page/) page shows....
- [**Summary**](/docs/browser/browser-monitoring/getting-started/browser-summary-page/) shows....